packages/core/src/utils.ts

```diff
@@ -243,3 +248,59 @@ export async function getRawConfiguration(
   }
   return sygmaConfig;
 }
+
+/**
+ * Validate Substrate address.
+ * @param {string} address
+ */
+export function isValidSubstrateAddress(address: string): boolean {
```

Comment:
since these functions return a `boolean` doesn't it make sense to return false instead of throwing errors

Reply:
good point, probably indeed with false it will be more useful

Reply:
But I will still throw an errors from the method isValidAddressForNetwork because as a user of SDK when I set the wrong destination address while creating the transaction I should see the error

Reply:
I think error should be thrown in a higher level SDK function

Reply:
@saadjhk Or probably I didn't get your suggestion

Reply:
a function that throw error can be `validateDestinationAddress` which is the one that throws an error and it uses functions like `isValidEthereumAddress`, `isValidSubstrateAddresss` which simply just return true or false

Reply:
@saadjhk I'm not completely getting your point. Here is the method I implemented, the only difference I throw individual error messages. Are you suggesting to use a flag variable "isValid" instead ?
```ts
/**
 * Validate Address based on network.
 * @param {string} address
 * @param {Network} network
 * @returns {boolean}
 */
export function isValidAddressForNetwork(address: string, network: Network): boolean {
  switch (network) {
    case Network.EVM:
      if (isValidEvmAddress(address)) return true;
      throw new Error('Invalid EVM Address');
    case Network.SUBSTRATE:
      if (isValidSubstrateAddress(address)) return true;
      throw new Error('Invalid Substrate Address');
    case Network.BITCOIN:
      if (isValidBitcoinAddress(address)) return true;
      throw new Error('Invalid Bitcoin Address');
    default:
      throw new Error('Provided network is not supported');
  }
}
```